metrics/file_length.py

```diff
@@ -25,10 +25,9 @@ def _parse(self, file, ast):
         )
         captures = query.captures(ast.root_node)
         count_comments = 0
-        for node in captures:
-            n = node[0]
+        for node, _ in captures:
```

Comment:
For my own understanding, why as an underscore been added to some  of the for loops in metrics?

Reply:
previously 'node' was a tuple of node, str. We do not need the string, so therefore it is _